src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -65,7 +65,7 @@ describe('XhrPlugin tests', () => {
                 statusText: 'OK'
             }
         });
-        expect(plugin.cacheSize).toEqual(0);
+        expect((plugin as any).xhrMap.size).toEqual(0);
```

Comment:
Note: type casting to `any` is how you access a private method/property in typescript